IDEAS/Buildings/Components/Interfaces/ZoneInterface.mo

```diff
@@ -95,7 +98,7 @@ initial equation
   for i in 1:nPorts loop
     assert(cardinality(ports[i])<=2,
       "Each element of ports should have zero or one external connections but " +
-      getInstanceName() +".ports[" + String(i) + "] has a different number." +
+      getInstanceName() +".ports[" + String(i) + "] has " + String(cardinality(ports[i]) - 1) + "." +
```

Comment:
@kldjonge did you remove this change? We need this for openmodelica compatibility.